### PR TITLE
Monitoring changes

### PR DESCRIPTION
**Pull request by punituee**
this is first phase of our api addition
limited validation and options have been specified
currently ListMetrics and GetMetricStatistics are supported

### CODE REVIEW

File: src/jcsclient/compute.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def start_instances(self, args):
         """
         return instance.start_instances(self.url, self.verb,
                                         self.headers, self.version, args)
-
```
Comment (anantkaushik89) [inline src/jcsclient/compute.py:150]: any reason for these changes?

File: src/jcsclient/utils.py [hunk 2]
```diff
@@ -158,6 +158,44 @@ def dash_to_camelcase(keyword):
         keyword += part
     return keyword
 
+def push_monitoring_indexed_params(params, key, vals):
```
Comment (anantkaushik89) [inline src/jcsclient/utils.py]: Isnt this function a copy of the function below it? Anything I am missing?

File: src/jcsclient/compute.py (hunk 1 above)
Comment (punituee) [inline src/jcsclient/compute.py:150]: the setting in my VIM editor shows trailing whitespaces in red. I have only removed them. 
The setting takes into regard a bare minimum indentation and white spaces check.

File: src/jcsclient/utils.py (hunk 2 above)
Comment (punituee) [inline src/jcsclient/utils.py]: yes anant see carefully, there is a change in parsing of data below, this does not remove the trailing 's' and adds a .member.<number>

**Pull request merged**